### ISSUE TITLE
[Bug]: Why can't I look up/down/left/right with WASD?

### ISSUE THREAD

**Issue by communist1917** [SuperTux/supertux#3083]
### SuperTux Version

0.6.3

### System Information

Manjaro Linux XFCE

### Expected Behavior

I expect looking up or down or left of right

### Actual Behavior

Pressing either W or A or S or D does nothing, view of level is not moving a pixel in any direction.

### Steps To Reproduce Actual Behavior

Press either W or A or S or D.

### Guidelines For Reporting Issues

- [X] I have read <https://github.com/SuperTux/supertux/blob/master/CONTRIBUTING.md#bug-reports>.
- [X] I have verified this isn't an issue that's already been reported.
- [X] I have verified this isn't a [discussion](https://github.com/SuperTux/supertux/discussions), or an issue about a crash or a feature request, but rather an actual bug ─ that is, the game did something not intended.
- [X] I have verified this issue is **not** about wrong translations (use Transifex for those), or anything unsupported (e.g. third-party add-ons).
- [X] In this report, I have only included details about **one** (1) bug.
- [X] If I make a mistake while submitting this report, I agree to use the "Edit" feature to correct it, instead of closing this issue and opening a new one.

**Comment by Frostwithasideofsalt** [comment]
You can already change the peek keybinds to WASD in the options.

**Comment by communist1917** [comment]
> You can already change the peek keybinds to WASD in the options.

Thank you.
In my options WASD buttons were already set as look buttons but they didn't work. Then I reset these buttons and set them again. And after I restrart Supertux they began working. This is strange.

**Comment by Rusty-Box** [comment]
Imo. WASD should be the default for peeking. Who uses "page-up", "delete", "home", etc. for games anyway? WASD is way more common in games to do something and is way more handy to use when the other hand is preoccupied with arrow keys, as well as being closer to Space and Ctrl. It just fit in better with all the other default key binds

**Comment by communist1917** [comment]
I figured out the problem.

If I set lookup buttons as "W", "A", "S", "D" and I launch Supertux while English keyboard layout is active then these buttons will work. But if I launch Supertux while Russian keyboard layout is active, these buttons WILL NOT work.

And vice versa. If I set lookup buttons as "Ц", "Ф", "Ы" "В" when I launch Supertux while Russian keyboard layout is active, these buttons will work. If launch the game while English keyboard layout is active, it WILL NOT work.

So, I think, Supertux algorithm compares key characters, not key codes.
